packages/terra-profile-image/tests/jest/ProfileImage.test.jsx

```diff
@@ -12,3 +12,9 @@ it('should render an avatar image without Source prop', () => {
   const wrapper = shallow(<ProfileImage alt="Default" height="75" width="75" />);
   expect(wrapper).toMatchSnapshot();
 });
+
+/* Profile Image with fit as contain */
+it('should render a Terra image with a placeholder prop', () => {
```

Comment:
The test description should be updated to describe the expectations of the test. It looks like this may have been copy pasted from above.

Reply:
Addressed here : https://github.com/cerner/terra-core/pull/2804/commits/87550e65421503ca5a0891c63b2619acaa3d2594